public/service_portal/v0.1.0/openapi.json

```diff
@@ -1160,6 +1203,29 @@
                     }
                 }
             },
+            "ServiceOfferingIcon": {
+                "type": "object",
+                "properties": {
+                    "id": {
+                        "type": "string",
+                        "title": "ID",
+                        "description": "The unique identifier for this Service Plan",
+                        "readOnly": true
+                    },
+                    "data": {
+                        "type": "string",
+                        "title": "Data",
+                        "description": "The raw SVG data for this icon",
+                        "readOnly": false
+                    },
+                    "source_ref": {
```

Comment:
Is `source_ref` returned with this object?

Reply:
Yes it is. the `/service_offering_icons` endpoint returns these exact 3 fields.